src/lib/CollectStore.js

```diff
@@ -284,8 +284,16 @@ export default class CollectStore {
     // detect oauth case
     const isOAuth = !!account.oauth
 
+    function createDirectoryIfNecessary (folderPath) {
+      if (folderPath) {
+        return cozy.client.files.createDirectoryByPath(folderPath)
+      } else {
+        return Promise.resolve({_id: null})
```

Comment:
I think that resolve with `null` is a better option than resolving with a mocked object which has no meaning. 
If the `_id` attribute is needed later we should adapt the code to not need it.
What do you think ?